intro-to-tests/automated-tests.md

```diff
@@ -66,21 +75,46 @@ Because there are a lot of different kinds of code, there are a lot of different
 
 ### Test Cases and Scenarios
 
-Testing a single function could have many unit tests to test it. Why? Based on the context, a single function could potentially have different arguments that give back different return values. Different scenarios in development can imply different arguments, which would produce different logic. All scenarios and possible outcomes of a function should be tested, as much as reasonably possible.
+Testing a single function will generally have many unit tests to test it. Why? Based on the context, a single function could potentially have different arguments that give back different return values. Different scenarios in development can imply different arguments, which would produce different logic. All scenarios and possible outcomes of a function should be tested, as much as reasonably possible.
 
 Addison could come up with the following scenarios:
 
 | Question to Ask                                                             | Specific Arguments                          | Expected Return Value       |
 | --------------------------------------------------------------------------- | ------------------------------------------- | --------------------------- |
-| Do we get an accurate birth chart if we pass in a date, location, and time? | `"January 1, 1980"`, `"Seattle"`, `"12:00"` | `{"Sun": "Capricorn", ...}` |
-| What happens if the function receives `None` for time?                      | `"January 1, 1980"`, `"Seattle"`, `None`    | `None`                      |
-| What happens if the `date` argument comes as a different format?            | `"1980-01-01"`, `"Seattle"`, `"12:00"`      | `{"Sun": "Capricorn", ...}` |
-| What happens if the function receives an empty string for `location`?       | `"January 1, 1980"`, `""`, `"12:00"` | should raise an exception   |
+| 1. Do we get an accurate birth chart if we pass in a date, location, and time? | `"January 1, 1980"`, `"Seattle"`, `"12:00"` | `{"Sun": "Capricorn", ...}` |
+| 2. What happens if the function receives `None` for time?                      | `"January 1, 1980"`, `"Seattle"`, `None`    | `None`                      |
+| 3. What happens if the `date` argument comes as a different format?            | `"1980-01-01"`, `"Seattle"`, `"12:00"`      | `{"Sun": "Capricorn", ...}` |
+| 4. What happens if the function receives an invalid place for `location`?       | `"January 1, 1980"`, `"xyz"`, `"12:00"` | should raise an exception   |
+
+To ensure that our test cases cover all scenarios, it can be helpful to categorize tests using the following categories:
+1. Positive Nominal
+1. Negative Nominal
+1. Positive Edge
+1. Negative Edge
```

Comment:
Up to your judgement, but I think we don't need to emphasize the positive and negative here; separation into nominal and edge cases feels sufficient.